includes/data-stores/class-wc-payment-token-data-store-table.php

```diff
@@ -0,0 +1,256 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Payment Token Data Store: Custom Table.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooThemes
+ */
+class WC_Payment_Token_Data_Store_Table implements WC_Payment_Token_Data_Store, WC_Object_Data_Store {
```

Comment:
O and don't rename existing ones before this PR since we'll all need rebase. Lets do that last.